lms/templates/dashboard/_dashboard_certificate_information.html

```diff
@@ -37,7 +37,7 @@
       <p class="message-copy">
         <%
           certificate_available_date_string = course_overview.certificate_available_date.strftime('%Y-%m-%dT%H:%M:%S%z')
-          container_string = _("Your certificate will be available on or after {date}")
+          container_string = _("Your grade and certificate will be ready after {date}.")
```

Comment:
Definitely my personal preference, but I find it a little confusing with the only saying "after" instead of "on".
Like the copy of `certificate will be ready after May 20, 2021.` makes me wonder does that mean I can't check until May 21? Why not just say the date I can actually check it?
But this might just be my weird brain. I don't know.

Reply:
This would be a UX question. I am going to bring this question back to Slack if that is ok.